README.md

```diff
@@ -2,208 +2,269 @@
+- `--source`/`-src`: Run against records from a specific source
+- `options`: Additional arguments passed as a list
 
-To build the container run the following command from the project root: `docker build -t drb-etl-pipeline .`
+Example commands:
 
-This will place an image `drb-etl-pipeline:latest` in your local docker instance. To run a process with the containerized application (in this instance the Flask API) execute the following command: `docker run drb-etl-pipeline -p APIProcess -e YOUR_ENV_FILE`. The `ENTRYPOINT` will accept the same arguments as invoking the process via the CLI.
+```bash
+# Run API in local environment
+python main.py -p APIProcess -e local
 
-When running a Docker image locally that interacts with other resources running on `localhost` it is necessary to supply a special URL to access them. Due to this it is generally helpful to define a unique `config` file for local docker testing. You may not wish to commit this file to git as it may contain secrets.
+# Import specific HathiTrust record
+python main.py -p HathiTrustProcess -e local -r record_id_123
 
-### Testing
+# Run custom import with limit
+python main.py -p NYPLProcess -e local -i custom -f my_records.csv -l 100
+```
 
-To run the unit tests, run `make unit`.
+You can also view all available flags by running:
 
-To run the integration tests, run `make integration`.
+```bash
+python main.py --help
+```
 
-To run the functional tests, run `make functional` (Your local development environment must be running in Docker for these functional tests to properly run)
+## Available Processes
 
-### Managing secrets
+The main processes available in this pipeline are:
 
-To keep sensitive settings out of git, some secrets configuration must be done to run the cluster. To set up for running on your local machine, copy the `config/example.yaml` file and provide the necessary configuration (ask a colleague if you need some of the keys required there). Then provide the name of this file as the `--environment` argument when you run scripts.
+Core Setup:
 
-Any file that contains sensitive details should not be committed to git. These values can be loaded via the AWS Parameter Store or AWS ECS. Speak to a NYPL engineer for access to these secrets and information on configuring your local setup to use them.
+- `LocalDevelopmentSetupProcess`: Initialize development database
+- `SeedLocalDataProcess`: Import sample data
+- `APIProcess`: Run the DRB API
+
+Data Ingestion:
+
+- `HathiTrustProcess`: Import from HathiTrust
+- `NYPLProcess`: Import from NYPL catalog
+- `GutenbergProcess`: Import from Project Gutenberg
+- `MUSEProcess`: Import from Project MUSE
+- `METProcess`: Import from MET Watson Digital Collections
+- `DOABProcess`: Import from Directory of Open Access Books
+- `LOCProcess`: Import from Library of Congress
+- `PublisherBacklistProcess`: Import from Publisher Backlist Project
```

Comment:
I do wonder if we want to maintain a list of these specifically since presumably, this list will change fairly often (?) and this feels likely to go stale. What do ya think of linking to https://github.com/NYPL/drb-etl-pipeline/tree/main/processes/ingest where these all live?